packages/react-charting/src/components/CommonComponents/CartesianChart.types.ts

```diff
@@ -486,7 +486,7 @@ export interface IModifiedCartesianChartProps extends ICartesianChartProps {
     isCalloutVisible: boolean;
     id: string;
     YValueHover?: IYValueHover[];
-    hoverXValue?: string | number | null;
+    hoverXValue?: string | number | Date | null;
```

Comment:
Is there no chart for which hoverXValue = Date does not work now

Reply:
Reverted this change